nestkernel/archiving_node.h

```diff
@@ -64,6 +65,36 @@ class Archiving_Node : public Node
    */
   Archiving_Node( const Archiving_Node& );
 
+  /**
+   * \fn double_t get_Ca_value()
+   * return the Ca_minus value
+   */
+  double_t
+  get_Ca_value() const
```

Comment:
Is this really the best method name? Generally, a getter for an attribute should be `get_<attr name>`, i.e., in this case `get_Ca_minus`. Is there any reason to do it differently?